doc/newsboat.asciidoc

```diff
@@ -890,8 +919,8 @@ This feature is often used to create a feed that contains all unread articles:
 
 	"query:Unread Articles:unread = \"yes\""
 
-Note the quotes that are necessary around the complete query "URL" and the
-backslashes that are necessary to escape the quotes in the filter expression.
+Note the <<_using_double_quotes,use of double quotes>> to preserve spaces in
```

Comment:
I'm a little bit in between. I really liked the two explicit hints on double quotes and their escapes, but this change makes it consistent with other paragraphs and even links to a much more detailed explanation. Let's keep it this way.